tests/dragonfly/replication_test.py

```diff
@@ -2014,7 +2014,7 @@ async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     df_factory.start_all([master, replica])
 
     c_replica = replica.client()
-    await c_replica.execute_command(f"DEBUG POPULATE 10000000")
+    await c_replica.execute_command(f"DEBUG POPULATE 8000000")
```

Comment:
this is a very slow test, takes lots of time. Borys should be working on fixing it (not by tuning the params)
hope to see a PR soon

Reply:
I still don't have an idea how to improve it without adding some debug commands additional info

Reply:
I think this test is not worth spending time on. Lets merge and move on.

Reply:
please mark this as a slow test and you can also add shutdown no save when stopping the replica to reduce the test time

Reply:
sorry, it was marked as automerge